Docs/Configuration.tex

```diff
@@ -2534,39 +2539,40 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
   \textbf{Requirement}: 10.14 (not required for older)\\
   \textbf{Description}: Set trim timeout in microseconds for APFS filesystems on SSDs.
 
-  APFS filesystem is designed in a way that the space controlled via
-  spaceman structure is either used or free. This may be different in
+  The APFS filesystem is designed in a way that the space controlled via
+  the spaceman structure is either used or free. This may be different in
   other filesystems where the areas can be marked as used, free, and
   \emph{unmapped}. All free space is trimmed (unmapped/deallocated)
   at macOS startup. The trimming procedure for NVMe drives happens
-  in LBA ranges due to the nature of \texttt{DSM} command with
+  in LBA ranges due to the nature of the \texttt{DSM} command with
   up to 256 ranges per command. The more fragmented the memory
-  on the drive is, the more commands are necessary to trim all the
-  free space.
-
-  Depending on the SSD controller and the drive fragmenation trim procedure
-  may take considerable amount of time, causing noticeable boot slowdown
-  APFS driver explicitly ignores previously unmapped areas and trims
-  them on boot again and again. To workaround boot slowdown macOS
-  driver introduced a timeout (\texttt{9.999999} seconds) that stops
-  trim operation when it did not manage to complete in time. On many
-  controllers, such as Samsung, where the deallocation is not very fast,
-  the timeout is reached very quickly. Essentially it means that
-  macOS will try to trim all the same lower blocks that have already been
-  deallocated, but will never have enough time to deallocate higher blocks
-  once the fragmentation increases. This means that trimming on these SSDs
-  will be broken soon after the installation, causing extra wear to the flash.
+  on the drive is, the more commands are necessary to trim
+  all the free space.
+
+  Depending on the SSD controller and the level of drive fragmenation, the trim
+  procedure may take a considerable amount of time, causing noticeable boot slowdown.
+  The APFS driver explicitly ignores previously unmapped areas and repeatedly trims
+  them on boot. To mitigate against such boot slowdowns, the macOS driver introduced a
+  timeout (\texttt{9.999999} seconds) that stops the trim operation when not completed in
```

Comment:
`when not finished in time.`?

Reply:
Isn't "finished" just a synonym here?
Doesn't add any nuance missing from "completed".

Reply:
don't see the difference either, could leave as is, unless @PMheart has a point

Reply:
Changed to "finished" to close out